### PR TITLE
[Ready for Review]: Added bundle install command

### PR DESCRIPTION
**Pull request by gkiar**
(sorry for not having template; creating PR from web)

- Added `bundle install` to commands that you need to run, at least the first time, to build the site successfully

### CODE REVIEW

**Comment by rainsworth** [comment]
Thanks @gkiar, pinging @alexmorley!

**Comment by alexmorley** [comment]
This is helpful for sure thanks so much @gkiar ! I put a comment on one of the commits. Additionally it could be useful to mention somewhere that ruby is required for installing locally?

**Comment by alexmorley** [comment]
Related to #550

File: book/README.md [hunk 1]
```diff
@@ -19,9 +19,10 @@ cd book/website
 pip install -r requirements.txt
 ```
 
-If you want to see your local changes on your own computer you'll have to go in the website directory and enter `make serve` (type `make` on it's own to see the other options).
+If you want to see your local changes on your own computer you'll have to go in the website directory, install the relevant ruby plugins, and enter `make serve` (type `make` on it's own to see the other options).
 ```
 cd book/website
+bundle install
```
[the remark this answers is not in the document]
Comment (gkiar) [reply book/README.md]: Sounds good, I didn't even think to look for an `install` mode in the `makefile`, derp... But yes, that's a better plan.

Comment (gkiar) [inline book/README.md]:
```suggestion
make install
```

**Comment by alexmorley** [comment]
LGTM

**Comment by alexmorley** [comment]
I can't fix the merge conflicts because I don't have access to this branch :(

**Comment by gkiar** [comment]
I made the fixes through Github's conflict-resolution manager, but for some reason it still says there are conflicts?

**Comment by alexmorley** [comment]
ship it :ship: :shipit:

**Comment by malvikasharan** [comment]
These changes have been integrated into the current version of README. Thank you, everyone.

**Pull request closed without merging**